bindings/java/src/org/sleuthkit/datamodel/SleuthkitJNI.java

```diff
@@ -1195,13 +1287,51 @@ public static void cancelFinishImage(long imgHandle) {
 	 *                          queried
 	 */
 	public static long findDeviceSize(String devPath) throws TskCoreException {
-		return findDeviceSizeNat(devPath);
+		getReadLock();
```

Comment:
I'm not sure this needs a lock.  This method is fully self contained and doesn't rely on any other handles.